omnibus/config/software/python3.rb

```diff
@@ -64,25 +65,66 @@
     block do
       FileUtils.rm_f(Dir.glob("#{install_dir}/embedded/lib/python#{major}.#{minor}/distutils/command/wininst-*.exe"))
     end
-  end
+  else
+    dependency "vc_redist_14"
+
+    vcrt140_root = "#{Omnibus::Config.source_dir()}/vc_redist_140/expanded"
+    ###############################
+    # Setup openssl dependency... #
+    ###############################
+    mkdir "externals/openssl-bin-3.0.15/amd64/include"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏
should this be kept in sync with the `openssl3` recipe?

Reply:
Not necessarily. That should match whatever the python build system expects, but as long as OpenSSL keeps a stable & backward compatible ABI & API (which they do) we can use any 3.x version we want